client/src/main/scala/com/aliyun/emr/rss/client/write/LifecycleManager.scala

```diff
@@ -271,6 +271,16 @@ class LifecycleManager(appId: String, val conf: RssConf) extends RpcEndpoint wit
         registerShuffleRequest.remove(shuffleId)
       }
       return
+    } else if (res.workerResource.isEmpty && numPartitions > 0) {
```

Comment:
Why the status is Success in this branch?

Reply:
> Why the status is Success in this branch?

RequestSlots rerun success but with empty slots.